README.md

```diff
@@ -18,6 +18,15 @@ have feedback you'd like to share with us.
 If you'd like to follow along, [CHANGELOG.md](CHANGELOG.md) is kept up to date
 with each commit.
 
+## Getting Started
+This readme assumes you have an environment configured with AWS credentials and IAM users/roles with appropriate access to QLDB. This readme focuses on shell configuration and commands. For a guide about prerequisites and general use of the shell, please see the developer guide: https://docs.aws.amazon.com/qldb/latest/developerguide/data-shell.html
+
+## QLDB Core Concepts
+QLDB provides guarantees about consistency during transactions, and ensures that transactions are consistent upon commit via Optimistic Currency Control. This applies to all queries, so when using the shell, your statements must be run as transactions. By default, the shell enables auto-commit mode, which will interpret queries by default as transactions, meaning you do not have to `start transaction` and `commit` manually each time. This is configurable (see below).
```

Comment:
Definitely need feedback on this language. I don't know that we clearly state anywhere about this behavior as it applies to interacting with the shell. I'm hoping one or two sentence reminder/refresher here will avoid any confusion or frustration when interacting with the shell.

I think we should also mention PartiQL, so I can at least link to the [PartiQL Reference](https://docs.aws.amazon.com/qldb/latest/developerguide/ql-reference.html).

I'll also update this to hyperlink [OCC](https://docs.aws.amazon.com/qldb/latest/developerguide/concurrency.html#concurrency.occ)

Any alternate proposals for language or what links to include would be greatly appreciated

Reply:
cc @bartolm

Reply:
Your proposed text sounds pretty clear and straightforward to me. Here's my suggestion for streamlining it a bit further, as well as some minor copyedits:

>QLDB ensures that transactions are consistent upon commit by using *optimistic currency control* (OCC). In QLDB, every statement (including every `SELECT` query) must run in a transaction.
>
>By default, the shell enables auto-commit mode. In this mode, the shell interprets each command that you enter as a separate PartiQL statement, meaning that you don't have to run `start transaction` and `commit` manually each time. This is configurable (see below).

Other dev guide topics we could consider linking are [Optimizing query performance](https://docs.aws.amazon.com/qldb/latest/developerguide/working.optimize.html) which also describes the 30-second transaction timeout limit, and [Session management with the driver](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-session-management.html) which details how QLDB sessions work and how the driver handles them.

Reply:
updating this, although I may leave out the session management aspect as I assume the shell makes this transparent